src/test/unit_tests/test_postgres_upgrade.js

```diff
@@ -0,0 +1,250 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+// setup coretest first to prepare the env
+// const coretest = require('./coretest');
+const mocha = require('mocha');
+const { Migrator } = require('../../upgrade/migrator');
+const assert = require('assert');
+const EventEmitter = require('events').EventEmitter;
+const _ = require('lodash');
+const postgres_client = require('../../util/postgres_client');
+const wtf = require('wtfnode');
+
+
+// coretest.setup({ pools_to_create: [], db_name: 'upgradetest' });
+process.env.CORETEST = 'upgrade-test';
+const COLLECTION_SIZE = 3000;
+// const COLLECTION_NUMBER = 100;
+const default_schema = {
+    id: 'default_schema',
+    type: 'object',
+    required: [
+        '_id',
+        'key1',
+        'key2',
+        'key3',
+    ],
+    properties: {
+        _id: { objectid: true },
+        key1: { type: 'string' },
```

Comment:
maybe use more types in the test other than string

Reply:
will do